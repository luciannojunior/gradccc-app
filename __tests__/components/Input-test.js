import 'react-native';
import React from 'react';
import Input from '../../src/components/Input';

import renderer from 'react-test-renderer';

it('Input renders correctly', () => {
  const tree = renderer.create(<Input />).toJSON();
  expect(tree).toMatchSnapshot();
});