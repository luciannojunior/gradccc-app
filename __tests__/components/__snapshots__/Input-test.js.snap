// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "height": 40,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "flex": 1,
        "fontSize": 18,
        "paddingLeft": 20,
      }
    }
  >
     
     
  </Text>
  <TextInput
    autoCorrect={false}
    onChangeText={undefined}
    placeholder={undefined}
    secureTextEntry={undefined}
    style={
      Object {
        "color": "#000",
        "flex": 2,
        "fontSize": 18,
        "lineHeight": 23,
        "paddingLeft": 5,
        "paddingRight": 5,
      }
    }
    underlineColorAndroid="transparent"
    value={undefined}
  />
</View>
`;
